src/Utils/Utils.php

```diff
@@ -155,28 +154,6 @@ public static function filter($iterable, callable $predicate): array
         return $assoc ? $result : array_values($result);
     }
 
-    /**
-     * @param iterable<mixed> $iterable
-     *
-     * @return array<mixed>
-     *
-     * @throws Exception
-     */
-    public static function map($iterable, callable $fn): array
```

Comment:
I'm very ok with dropping this. But `array_map()` should be almost of the same speed as foreach in PHP 8 and sometimes `array_map()` is more readable IMO

https://stackoverflow.com/a/64460563/519333

Reply:
As a foundational library, I think it is a good choice for us to forgo readability in favor of performance when in question. It is nice to see that the speed of abstractions is catching up. If there was a generic `map` which performed better for all `iterable`, I would be all for it.

Reply:
Yup, dropping this utility is just fine but e.g. this https://github.com/webonyx/graphql-php/pull/712/files#diff-17372694d53aa9749dbda064d15720cf6fb7df4f071c97c26d8d692a66d39a9bL205-L208 I'd keep because the performance should be the same on 8.0 already and is more readable.

Reply:
Restored some of the short `array_map`.